src/sentry/integrations/slack/link_identity.py

```diff
@@ -40,24 +39,7 @@ class SlackLinkIdentityView(BaseView):
     def handle(self, request, signed_params):
         params = unsign(signed_params.encode("ascii", errors="ignore"))
 
-        try:
-            organization = Organization.objects.get(
-                id__in=request.user.get_orgs(), id=params["organization_id"]
-            )
-        except Organization.DoesNotExist:
-            raise Http404
-
-        try:
-            integration = Integration.objects.get(
-                id=params["integration_id"], organizations=organization
-            )
-        except Integration.DoesNotExist:
-            raise Http404
-
-        try:
-            idp = IdentityProvider.objects.get(external_id=integration.external_id, type="slack")
-        except IdentityProvider.DoesNotExist:
-            raise Http404
+        organization, integration, idp = getIdentity(request, params)
```

Comment:
@NisanthanNanthakumar Python functions should be camel_case btw. I think the first arg here should be just the `user` since that's all we use of the request. It probably makes sense to use ordered parameters here instead of passing a dictionary:
`def get_identity(user, integration_id, organzation_id)`
It makes it more clear what the expected inputs are